### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rtviii**
Added the Solana RPC implementation in Go to the Blockchains section. This one's going to be important ;)

### CODE REVIEW

**Comment by avelino** [comment]
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body links content check:
- godoc.org or pkg.go.dev: True
- goreportcard.com: False
- coverage: False

> your project is about to be approved, it's under revision, _it may take a few days_

**Review by phanirithvij — CHANGES_REQUESTED** [review]
Please follow the pull request template after reading the contribution guidelines https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standards

**Review by Khalid55511 — APPROVED** [review]

**Comment by phanirithvij** [comment]
@rtviii ping.

**Comment by rtviii** [comment]
@phanirithvij hey, sorry. moved to the alphabetical order and added appropriate links! take a look, thanks!!:)

File: README.md [hunk 1]
```diff
@@ -189,7 +189,8 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 
 * [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) - A Framework for Building Public Blockchains in the Cosmos Ecosystem.
 * [go-ethereum](https://github.com/ethereum/go-ethereum) - Official Go implementation of the Ethereum protocol.
-* [gossamer](https://github.com/ChainSafe/gossamer) - A Go implementation of the Polkadot Host.
+* [gossamer](https://github.com/ChainSafe/gossamer) - A Go implementation of the Polkadot Host
+* [solana-go](https://github.com/gagliardetto/solana-go) - Go library to interface with Solana JSON RPC and WebSocket interfaces.
```
Comment (phanirithvij) [inline README.md:193]: - Golint has many exported functions with no comments
- No coverage report in tests, and I see lots of `no test files` in the repo's test GitHub action.

- I'll see if I can test the coverage locally in a few hours

**Review by phanirithvij — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -189,7 +189,8 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 
 * [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) - A Framework for Building Public Blockchains in the Cosmos Ecosystem.
 * [go-ethereum](https://github.com/ethereum/go-ethereum) - Official Go implementation of the Ethereum protocol.
-* [gossamer](https://github.com/ChainSafe/gossamer) - A Go implementation of the Polkadot Host.
+* [gossamer](https://github.com/ChainSafe/gossamer) - A Go implementation of the Polkadot Host
```
Comment (phanirithvij) [inline README.md:192]:
```suggestion
* [gossamer](https://github.com/ChainSafe/gossamer) - A Go implementation of the Polkadot Host.
```

**Review by Khalid55511 — APPROVED** [review]

**Review by phanirithvij — APPROVED** [review]

**Pull request merged**